com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Photoshop.md

```diff
@@ -1,62 +1,67 @@
 # Authoring lookup textures with Adobe Photoshop
 
-The following process uses Photoshop, but you can apply it for other digital content creation tools such as [DaVinci Resolve](LUT-Authoring-Resolve.md) or Nuke.
+The following process uses Photoshop, but you can apply it to other digital content creation tools such as [DaVinci Resolve](LUT-Authoring-Resolve.md) or Nuke.
 
 ## Step 1: Set up the HDRP Asset
 
-Decide on the lookup texture (LUT) size to use for the Project, and set it in your Project's [HDRP Asset](HDRP-Asset.md). To set the LUT size, use the HDRP Asset’s **Grading LUT Size** property.
+Decide on the lookup texture (LUT) size to use for the Project, and set it in your Project's [HDRP Asset](HDRP-Asset.md). To set the LUT size, use the HDRP Asset’s **Grading LUT Size** property. To find this property:
 
-This is a fixed size; if you change it later, you need to rebake all of your LUTs. A size of **32** provides enough precision for real-time use.
+1. Open your HDRP Asset in the Inspector window.
+2. Go to **Post-processing** > **Grading LUT Size**
+
+This is a fixed size; if you change it later, you need to rebake all your LUTs. A size of **32** provides enough precision for real-time use.
 
 ## Step 2: Render the Camera view to an EXR file
 
 Export the current [Camera](HDRP-Camera.md)'s view to a log-encoded EXR file. To do this:
 
 1. In the Scene view or Hierarchy, select a GameObject that contains a Camera component.
-2. Select **Edit** > **Rendering** > **Render Selected HDRP Camera to Log EXR** **(**or press **Ctrl+Shift+Alt+E)**.
```

Comment:
Any reasons to remove the keyboard shortcut? this is useful for people

Reply:
hmm I notice doc is very simialr to: https://github.com/Unity-Technologies/Graphics/pull/6995/files#diff-25001ab39fff1c541102518bd4967fb93681446d0287d30e525aad8ba519d5ccL16 . Not sure why we have two page here, we should share stuff

Reply:
I've addressed these comments on the other PR